docs/changelog.rst

```diff
@@ -238,6 +268,17 @@ Other:
 * Update docs and noxfile.
   `#5071 <https://github.com/pybind/pybind11/pull/5071>`_
 
+Version 2.12.1 (September 13, 2024)
+-----------------------------------
+
+New Features:
+
+* A new ``self._pybind11_conduit_v1_()`` method is automatically added to all
```

Comment:
Maybe mention "Backport" here somehow?

(Only if you think that's helpful.)

Reply:
Added:  Supported on pybind11 2.11.2, 2.12.1, and 2.13.6+.